Mathlib/Tactic/FunProp/Continuous.lean

```diff
@@ -1,226 +0,0 @@
-
-theorem ContinuousAt.div₀ (hf : ContinuousAt f a) (hg : ContinuousAt g a) (h₀ : g a ≠ 0) :
-    ContinuousAt (fun x => f x / g x) a := ContinuousAt.div hf hg h₀
-
-theorem ContinuousOn.div₀ (hf : ContinuousOn f s) (hg : ContinuousOn g s) (h₀ : ∀ x ∈ s, g x ≠ 0) :
-    ContinuousOn (fun x => f x / g x) s := ContinuousOn.div hf hg h₀
-
-
-end div
-
-end Missing
-
-
-
--- mark definition
-attribute [fun_prop]
-  Continuous
-  ContinuousAt
-  ContinuousOn
-
-
--- lambda rules
-attribute [fun_prop]
-  continuous_id'
-  continuous_const
-  Continuous.comp'
-  continuous_apply
-  continuous_pi
-
-  continuousAt_id'
-  continuousAt_const
-  ContinuousAt.comp'
-  continuousAt_apply
-  continuousAt_pi'
-
-  continuousOn_id'
-  continuousOn_const
-  ContinuousOn.comp'
-  continuousOn_apply
-  continuousOn_pi'
-
--- product
-attribute [fun_prop]
-  Continuous.prod_mk
-  Continuous.fst
-  Continuous.snd
-
-  ContinuousAt.prod
-  ContinuousAt.fst
-  ContinuousAt.snd
-
-  ContinuousOn.prod
-  ContinuousOn.fst
-  ContinuousOn.snd
-
--- transitions
-attribute [fun_prop]
-  Continuous.continuousAt
-  Continuous.continuousOn
-
--- algebra
-attribute [fun_prop]
-  Continuous.add
-  Continuous.sub
-  Continuous.neg
-  Continuous.pow
-  Continuous.zpow
-  Continuous.zpow₀
-  Continuous.mul
-  Continuous.smul
-  Continuous.const_smul
-  Continuous.vadd
-  Continuous.const_vadd
-  Continuous.div'
-  Continuous.div₀
-  Continuous.inv
-  Continuous.inv₀
-  Continuous.star
-  Continuous.sup
-  Continuous.inf
-  Continuous.abs
-
-  Continuous.max
-  Continuous.min
-
-  ContinuousAt.add
-  ContinuousAt.sub
-  ContinuousAt.neg
-  ContinuousAt.pow
-  ContinuousAt.zpow
-  ContinuousAt.zpow₀
-  ContinuousAt.mul
-  ContinuousAt.smul
-  ContinuousAt.const_smul
-  ContinuousAt.vadd
-  ContinuousAt.const_vadd
-  ContinuousAt.div'
-  ContinuousAt.div₀
-  ContinuousAt.inv
-  ContinuousAt.inv₀
-  ContinuousAt.star
-  ContinuousAt.sup
-  ContinuousAt.inf
-  ContinuousAt.abs
-
-  ContinuousOn.add
-  ContinuousOn.sub
-  ContinuousOn.neg
-  ContinuousOn.pow
-  ContinuousOn.zpow
-  ContinuousOn.zpow₀
-  ContinuousOn.mul
-  ContinuousOn.smul
-  ContinuousOn.const_smul
-  ContinuousOn.vadd
-  ContinuousOn.const_vadd
-  ContinuousOn.div'
-  ContinuousOn.div₀
-  ContinuousOn.inv
-  ContinuousOn.inv₀
-  ContinuousOn.star
-  ContinuousOn.sup
-  ContinuousOn.inf
-  ContinuousOn.abs
-
--- analysis
-attribute [fun_prop]
-  Continuous.dist
-  Continuous.nndist
-  Continuous.edist
-
-  Continuous.norm
-  Continuous.nnnorm
-  Continuous.norm'
-  Continuous.nnnorm'
-
-  ContinuousAt.norm
-  ContinuousAt.nnnorm
-  ContinuousAt.norm'
-  ContinuousAt.nnnorm'
-
-  ContinuousOn.norm
-  ContinuousOn.nnnorm
-  ContinuousOn.norm'
-  ContinuousOn.nnnorm'
-
--- special function
-attribute [fun_prop]
-  Continuous.exp
-  Continuous.cexp
-  Continuous.log
-  Continuous.pow
-  Continuous.sqrt
-
-  ContinuousAt.exp
-  ContinuousAt.cexp
-  ContinuousAt.log
-  ContinuousAt.pow
```

Comment:
Is this the same lemma that is also tagged earlier?

Reply:
Yes, it was accidentally tagged twice.